code/mathematical_algorithms/src/add_polynomials/add_polynomials.c

```diff
@@ -16,8 +16,7 @@ struct Node
 /*
  * Function to create new node.
  */
-void 
-create_node(int x, int y, struct Node **temp)
+void create_node(int x, int y, struct Node **temp)
```

Comment:
The older version was correct; The C style guide says the type goes on its own line.